tests/views/test_general_site_metrics_view.py

```diff
@@ -18,20 +18,18 @@
 
 def mock_get_monthly_site_metrics(date_for=None, **kwargs):
     return dict(
-        monthly_active_users=1,
-        total_site_users=2,
         total_site_learners=3,
         total_site_staff_users=4,
         total_site_coures=5,
-        total_course_enrollments=6,
-        total_course_completions=7,
         total_active_courses=8,
     )
 
+
 @pytest.mark.django_db
 class TestGeneralSiteMetricsView(BaseViewTest):
-    '''Tests the GeneralSiteMetricsView view class
-    '''
+    """
+    Tests the GeneralSiteMetricsView view class
```

Comment:
GeneralSiteMetricsView -> "GeneralSiteMetricsView"

Reply:
@taimoor-ahmed-1 this core change is not needed. I am reverting it.